tests/unit/test_finder.py

```diff
@@ -276,6 +278,37 @@ def test_link_sorting_wheels_with_build_tags(self):
         results2 = sorted(reversed(links), key=sort_key, reverse=True)
         assert links == results == results2, results2
 
+    def test_build_tag_is_less_important_than_other_tags(self):
+        links = [
+            InstallationCandidate(
+                "simple",
+                "1.0",
+                Link('simple-1.0-1-py3-abi3-linux_x86_64.whl'),
+            ),
+            InstallationCandidate(
+                "simple",
+                '1.0',
+                Link('simple-1.0-2-py3-abi3-linux_i386.whl'),
+            ),
+            InstallationCandidate(
+                "simple",
+                '1.0',
+                Link('simple-1.0.tar.gz'),
+            ),
+        ]
+        valid_tags = [
+            Tag('py3', 'abi3', 'linux_x86_64'),
+            Tag('py3', 'abi3', 'linux_i386'),
+        ]
+        evaluator = CandidateEvaluator(
+            'my-project', supported_tags=valid_tags, specifier=SpecifierSet(),
+        )
+        sort_key = evaluator._sort_key
+        results = sorted(links, key=sort_key, reverse=True)
+        results2 = sorted(reversed(links), key=sort_key, reverse=True)
```

Comment:
I would compare the result to a static list instead. The test now is basically “CandidateEvaluator uses `sort_key` to sort”, which does not really mean anything. The thing to test should be “CandidateEvaluator sorts things exactly like this” instead.

Reply:
Read the following line again. The test does compare the result to a static list. The static list is called `links`.
```py
assert links == results == results2, results2
```
perhaps this is too confusing. Should I rewrite it as below?
```py
assert links == results, results
assert links == results2, results2
```

Reply:
Yea, that would be a good idea! :)

Reply:
done